docs/source/development/adding_to_the_api_documentation.ipynb

```diff
@@ -71,7 +70,10 @@
     "    callback: Optional[Callable] = None,\n",
     "    **kwargs\n",
     ") -> Tuple[\n",
-    "    \"probnum.RandomVariable\", \"probnum.RandomVariable\", \"probnum.RandomVariable\", Dict\n",
+    "    \"randvars.RandomVariable\",\n",
+    "    \"randvars.RandomVariable\",\n",
+    "    \"randvars.RandomVariable\",\n",
+    "    Dict,\n",
     "]:\n",
     "    \"\"\"Infer a solution to the linear system :math:`A x = b` in a Bayesian framework.\n",
```

Comment:
...just pointing out this formulation reg. the `solve/infer_solution` discussion :laughing: